docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -140,14 +140,19 @@ Allows setting a sort order for the result. Defaults to `start_time`, i.e. sorti
 (Optional, string)
 Sort order. Valid values are `asc` for ascending and `desc` for descending order. Defaults to `asc`, meaning ascending order.
 
+`after_value`::
+(Optional, string)
+Value of the current sort column at which to start pagination. Can either be a string snapshot- or repository name when sorting by
```

Comment:
I think pagination is not the right word here, since it is just where you start retrieval of data from?

```suggestion
Value of the current sort column at which to start retrieval. Can either be a string snapshot- or repository name when sorting by
```